lib/presentation/widgets/word_cloud.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_scatter/flutter_scatter.dart';
+
+import '../../core/theme/app.dart';
+import '../models/word_cloud/word_cloud_data.dart';
+
+class WordCloudExample extends StatelessWidget {
+  const WordCloudExample({Key? key}) : super(key: key);
+
```

Comment:
Add a parameter `final List<Keyword> data;` here and in the constructor.
So that way we can inject the data into this widget. that way it will be easier to add/remove mock data and inject real data from parent widget using a Bloc. You can have a look at the class `SentimentChart`

Reply:
Add a `static const keyPrefix = 'WordCloud';` which would be used to define keys of its children widget. This is for units tests.  You can have a look at `SentimentChart` for an example

Reply:
Addressed in commit 18c300f2224c821f3449b96545a2ea732c7dc358